pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -869,6 +869,8 @@ type GcsStorageProvider struct {
 	// SecretName is the name of secret which stores the
 	// gcs service account credentials JSON .
 	SecretName string `json:"secretName"`
+	// Prefix for the keys.
+	Prefix string `json:"prefix,omitempty"`
```

Comment:
Plz provide some note comments to explain the `Prefix` here.

Reply:
done